assets/css/style.css

```diff
@@ -264,6 +264,72 @@ header {
     margin-top: 100px;
     text-align: justify;
 }
+.footer {
+    background-color: #FFE66D;
+    background-size: cover;
+    padding :30px 40px;
+}
+.footer-logo {
+    color: #0e2d31;
+    font-size: 60px;
+    float: right;
+    padding-right: 100px;
+}
+
+.footer-list {
+    color: #1a535c;
+    font-size: 25px;
+}
+.footer-list li{
+    list-style: none;
+    padding-right: 60px;
+    padding-left: 20px;
+    cursor: pointer;
+}
+.Beranda {
+    float: left;
+
+}
+.Gejala {
+    float: left;
+}
+.Pencegahan {
+    float: left;
+}
+.call-center{
+    text-align: right;
+    font-size: 22px;
+    padding-top: 30px;
+    margin-top : 20px;
+    padding-right: 70px;
+    padding-bottom: 10px;
+    color:#0e2d31;
+    
+}
+.copyright{
+    font-size: 13px;
+    color: #0e2d31;
+    opacity: 0.8;
+    padding-left: 30px;
+    border-top: 1px solid rgba(14,45,49,0.8);
+    margin-top: 10px;
+}
+.copyright span {
+    border-left: 1px solid rgba(14,45,49,0.8);
+    padding-left: 10px;
+    margin-left: 10px;
+}
+.flex-container {
+    display: flex;
+    flex-direction : column;
+    padding-top: 30px;
+}
+.flex-container {
+    display: flex;
+    flex-direction : column;
+    flex-wrap: wrap;
+    padding-top: 45px;
+}
```

Comment:
See line 322 to 332. You have duplicate class `flex-container`.

Remove one of them and change the class name, for example to `footer-container`. Make sure to change the class used in index.html too.

assets/css/style.css
```
.footer-container {
    display: flex;
    flex-direction : column;
    flex-wrap: wrap;
    padding-top: 45px;
}
```

index.html
```
        <footer class="footer">
            <div class="footer-logo">InCov</div>
            <div class="footer-list"> 
            <div class="footer-container">
                <ul>
                    <li href="#" class="Beranda"> Beranda </li>
                    <li href="#" class="Gejala"> Gejala </li>
                    <li href="#" class="Pencegahan"> Pencegahan </li>
                    <li href="#" class="Jenis-Pasien"> Jenis Pasien </li>
                </ul>
            </div>
...
```